fmralign/alignment_methods.py

```diff
@@ -267,7 +269,9 @@ def fit(self, X, Y):
             target data
         """
         self.R = RidgeCV(alphas=self.alphas, fit_intercept=True,
-                         normalize=False, scoring=None, cv=self.cv)
+                         normalize=False,
+                         scoring=sklearn.metrics.SCORERS['r2'],
```

Comment:
Why did you change the scoring method ? I'm not sure what the default meant.

Reply:
I changed the scoring method because it was giving a `FutureWarning`

<details>

> /srv/conda/envs/notebook/lib/python3.7/site-packages/sklearn/base.py:434: FutureWarning: The default value of multioutput (not exposed in score method) will change from 'variance_weighted' to 'uniform_average' in 0.23 to keep consistent with 'metrics.r2_score'. To specify the default value manually and avoid the warning, please either call 'metrics.r2_score' directly or make a custom scorer with 'metrics.make_scorer' (the built-in scorer 'r2' uses multioutput='uniform_average').

</details>

Now it should be using the [r2_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html) metric directly, so the `multioutput` default is `uniform_average` rather than `variance_weighted` !